homeassistant/components/velbus/__init__.py

```diff
@@ -106,11 +106,12 @@ def __init__(self, channel):
     @property
     def unique_id(self):
         """Get unique ID."""
-        return "{}.{}.{}".format(
-            self._channel.get_module_type(),
-            self._channel.get_module_address(),
-            self._channel.get_channel_number(),
-        )
+        serial = 0
+        if self._channel.get_module_serial() == 0:
+            serial = self._channel.get_module_address()
+        else:
+            serial = self._channel.get_module_serial()
```

Comment:
```suggestion
        if serial:= self._channel.get_module_serial() == 0:
            serial = self._channel.get_module_address()
```

Reply:
i tried using this syntax, but the flake8 does not really like this

`flake8...................................................................Failed
- hook id: flake8
- exit code: 1

homeassistant/components/velbus/__init__.py:138:19: E701 multiple statements on one line (colon)
homeassistant/components/velbus/__init__.py:138:19: E231 missing whitespace after ':'
homeassistant/components/velbus/__init__.py:138:20: E225 missing whitespace around operator
homeassistant/components/velbus/__init__.py:138:20: E999 SyntaxError: invalid syntax
`